### ISSUE TITLE
Abuse prevention

### ISSUE THREAD

**Issue by southrop** [southrop/seiyuu#10]
Probably need to think about this some time.
- Throttling
- Input sanitation (probably not necessarily as we don't directly do anything with the input, instead we match against it)
- Ignore other bots

**Comment by omonomono** [comment]
Need to disable tweeting the @ symbol to avoid looping with other bots, if you allow user input from a spreadsheet.

**Comment by southrop** [comment]
Nah, the bot will never pull directly from the spreadsheet, so I think we're fine on that aspect.

**Comment by omonomono** [comment]
Well, not sure what you mean by that so I don't know how it removes the possibility of the bot getting into a tweet loop, but as long as you understand the possible issue.

**Comment by southrop** [comment]
As in I will always have full control over the list of possible things it can tweet. A bigger concern might be someone mentioning a bunch of bots together. Perhaps we should ignore tweets that mention more users than just the bot.

**Comment by omonomono** [comment]
That's a safe design. So is an actual rate limit.
